src/suites/cts/copyImageBitmapToTexture.spec.ts

```diff
@@ -0,0 +1,83 @@
+export const description = `
+copy imageBitmap To texture tests.
+`;
+
+import { TestGroup } from '../../framework/index.js';
+
+import { GPUTest } from './gpu_test.js';
+
+export const g = new TestGroup(GPUTest);
+
+function calculateRowPitchForImageBitmap(width: number, bytesPerPixel: number): number {
+  const bytesPerRow = width * bytesPerPixel;
+  if (bytesPerRow % 256 === 0) {
+    return bytesPerRow;
+  }
+  return bytesPerRow + 256 - (bytesPerRow % 256);
+}
+
+g.test('from image element', async t => {
+  const img = document.createElement('img');
+  img.src = '../../assets/img/test.png';
+  await img.decode();
+
+  const imageBitmap = await createImageBitmap(img, 0, 0, img.width, img.height);
```

Comment:
If you remove the extra parameters here, I think the img.decode shouldn't be needed.

```suggestion
  const imageBitmap = await createImageBitmap(img);
```

Reply:
That seems great!

Reply:
Done